.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: publish
```

Comment:
So I’ve always been unhappy with UI’s npm publishing with a custom script.

The first way to simplify things is to do what many other npm packages do: always use `—tag latest` for releases, and `--tag next` for prereleases. Trying to parse the tag from Git may be excessive and we haven’t exactly needed it (for example, if one needs a custom tag they can publish it manually)

So anyways: first simplification is probably having separate `release` and `prerelease` workflows that force the tag to be either `latest` or `next`

Reply:
And for clarity: GitHub Actions knows to run either based on whether the tag is `v0.0.1` or `v0.0.1-rc.0` format